near-sdk/src/json_types/public_key.rs

```diff
@@ -135,12 +150,12 @@ impl<'de> serde::Deserialize<'de> for Base58PublicKey {
 
 impl From<&Base58PublicKey> for String {
```

Comment:
`From<X> for String` is an anti-pattern (unless the internal impl is a `String`), better to implement `fmt::Display`

Reply:
Wouldn't this force us to use `format!` which potentially adds to the code size? The impl for `serde::Serialize` makes use of into String so we would end having to call `format!` instead there